src/components/Form.jsx

```diff
@@ -0,0 +1,98 @@
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+import { TextField } from '@mui/material';
+import { Button } from '@mui/material';
+import styles from './Form.module.scss';
+import { useState } from 'react';
+import axios from 'axios';
+
+const validationSchema = Yup.object().shape({
+  title: Yup.string().required('This field is required'),
+  price: Yup.string().required('This field is required'),
+  imageUrl: Yup.string().required('This field is required'),
+});
+function Form({ onCloseForm }) {
+  //   const [addProduct, setAddProduct] = useState(false);
+
+  //   const onClickAddProduct = async () => {
+  // try {
+  //   const respons= await axios.post(
+  //     'https://6327175fba4a9c47533089b9.mockapi.io/items/',{
+  //       title:'',
+  //       price:'',
+  //       imageUrl:''
+  //     })
+  //     alert(respons.data.messsage)
+  // } catch (error) {
+  //   alert('e')
+  // }
+
+  //     setAddProduct(true);
+  //   };
+
+  const formik = useFormik({
+    initialValues: {
+      title: '',
+      price: '',
+      imageUrl: '',
+    },
+    validationSchema: validationSchema,
+    onSubmit: (values) => {
+      onCloseForm();
+      axios.post('https://6327175fba4a9c47533089b9.mockapi.io/items/', values);
+    },
+  });
+
+  return (
+    <div>
+      <form
+        onSubmit={formik.handleSubmit}
+        className={styles.form}
+      >
+        <TextField
```

Comment:
Please create reusable TextField component and use textFieldHelper.map to render all fields